src/main/app/claims/claimStoreClient.ts

```diff
@@ -44,7 +44,7 @@ export default class ClaimStoreClient {
       })
   }
 
-  static retrieveByLetterHolderId (letterHolderId: number): Promise<Claim> {
+  static retrieveByLetterHolderId (letterHolderId: string): Promise<Claim> {
```

Comment:
why is this now string?

Reply:
number was an implementation detail and is soon going away, I'm working on a PR to clean it up more but that will be separate to this